qml/CredentialCardIcon.qml

```diff
@@ -1,18 +1,143 @@
 import QtQuick 2.9
 import QtQuick.Controls 2.2
 import QtQuick.Controls.Material 2.2
+import "utils.js" as Utils
 
 Rectangle {
 
-    property string letter: "A"
+    property string letter: getIconLetter()
+    property var _credential
```

Comment:
Solved it by just removing the property, since `credential` is always available assuming the `CredentialCardIcon` is the child of a `CredentialCard` (which it always is).